script/brakeman

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+# Does not install the latest brakeman if already installed
+gem install brakeman --conservative
+ruby -rbrakeman -S rake -f lib/tasks/brakeman.rake brakeman
```

Comment:
`ruby -rbrakeman -S` is to ensure the command is run without bundled gems. e.g. rubygems-bundler installed by rvm would otherwise cause this script to fail.